keen/tests/client_tests.py

```diff
@@ -8,20 +8,37 @@
 import keen
 from keen.client import KeenClient
 from keen.tests.base_test_case import BaseTestCase
+from mock import patch, MagicMock
 import sys
 
 __author__ = 'dkador'
 
 
+class MockedRequest(object):
+    def __init__(self, status_code, json_response):
+        self.status_code = status_code
+        self.json_response = json_response
+
+    def json(self):
+        return {"result": self.json_response}
+
+
+class MockedFailedRequest(MockedRequest):
```

Comment:
Since you only use this with 401 errors, you could do something like:

```
class MockedFailedRequest(object):
    def __init__(self, failure_response):
        self.status_code = 401
        self.failure_response = failure_response

    def json(self):
        return failure_response
```